lib/utils/containsString.js

```diff
@@ -28,15 +30,15 @@ module.exports = function containsString(
 };
 
 function testAgainstString(value, comparison) {
-	if (!comparison) return false;
-
-	if (comparison[0] === '/' && comparison[comparison.length - 1] === '/') {
+	if (
+		!comparison ||
+		_.isRegExp(comparison) ||
+		(comparison.startsWith('/') && comparison.endsWith('/'))
+	) {
 		return false;
 	}
 
 	if (value.includes(comparison)) {
 		return { match: value, pattern: comparison };
 	}
-
-	return false;
```

Comment:
I think this one is needed if every condition before fails.

Reply:
The way I have the patch, this wouldn't be reached.